src/org/usfirst/frc/team1518/robot/Robot.java

```diff
@@ -84,33 +75,48 @@ public Robot() {
     /** Autonomous operations
      * 
      */
-    public void autonomous() {
+    public void autonomousInit() {
+        mode = (int) autoChoice.getSelected();
```

Comment:
Like here, we'd have to loop through the buttons to see which one was turned on so we know which mode to use.